### PR TITLE
Report QPS about each type of command in RegionHeartbeat 

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

To solve #2526

### Check List

Tests

- Unit test

Related changes

- PR to update [`pingcap/kvproto`](https://github.com/pingcap/kvproto/pull/638)

### Release note

- No release note

### CODE REVIEW

File: server/core/region.go [hunk 1]
```diff
@@ -403,6 +403,14 @@ func (r *RegionInfo) GetReplicationStatus() *replication_modepb.RegionReplicatio
 	return r.replicationStatus
 }
 
+// GetReadQPS returns the region's qps by type.
+func (r *RegionInfo) GetReadQPS() uint64 {
+	if r.queryStats == nil {
+		return 0
+	}
+	return r.queryStats.Get + r.queryStats.Coprocessor + r.queryStats.Scan
```
Comment (disksing) [inline server/core/region.go:411]: do we need to assign different weights for different query types? As that `coprocessor` or `scan` requests can be heavier than point gets.

File: server/core/region.go [hunk 2]
```diff
@@ -403,6 +403,22 @@ func (r *RegionInfo) GetReplicationStatus() *replication_modepb.RegionReplicatio
...
+}
+
+// GetQPSWrite returns the region's write QPS.
+func (r *RegionInfo) GetQPSWrite() uint64 {
+	if r.queryStats == nil {
+		return 0
+	}
+	return r.queryStats.Put
```
Comment (disksing) [inline server/core/region.go:419]: Should `Delete` be considered as write too?

**Comment by lhy1024** [comment]
PTAL @HunDunDM

File: server/core/region.go (hunk 1 above)
Comment (ZenoTan) [reply server/core/region.go:427]: Or we can have a better compare function when ranking topK?

Comment (lhy1024) [reply server/core/region.go:427]: Do you have any good suggestions

**Pull request closed without merging**